supervisor/host/const.py

```diff
@@ -14,6 +14,13 @@ class InterfaceMethod(str, Enum):
     AUTO = "auto"
 
 
+class InterfaceAddrGenMode(str, Enum):
```

Comment:
```suggestion
class InterfaceAddrGenMode(StrEnum):
```
After you rebase this suggestion will become more obvious, all the other ones are changed this way as well. `StrEnum` is the new and improved way of making an Enum where every instances is also a string added in 3.11 as opposed to inheriting from both like you used to have to do.